scripts/exregional_novarcldana.sh

```diff
@@ -0,0 +1,315 @@
+#!/bin/bash
```

Comment:
The name of this script uses yet another abbreviation of "analysis". Please consider renaming it to use "anl" for clarity and consistency. I'd suggest the same for the executable name, too.

Reply:
Changed from cldana to cldanl.